scripts/docker-compose/imports/smartdocuments-wiremock/mappings/wsxmldeposit-wizard.json

```diff
@@ -1,75 +1,127 @@
+                  "uiterlijkeEinddatumAfdoening":"${json-unit.any-string}",
+                  "verlengingReden":"fakeReason",
+                  "vertrouwelijkheidaanduiding":"openbaar",
+                  "zaaktype":"Melding evenement organiseren behandelen"
+                }
+              }
             },
-            "productAanvraagtype": "productaanvraag-type-1"
+            "ignoreArrayOrder":true
           },
-          "zaak": {
-            "communicatiekanaal": "E-formulier",
-            "groep": "Test group A",
-            "identificatie": "ZAAK-1970-0000000001",
-            "omschrijving": "fakeZaakOmschrijving",
-            "registratiedatum": "${json-unit.any-string}",
-            "startdatum": "01-01-1970",
-            "status": "Wacht op aanvullende informatie",
-            "toelichting": "Aangemaakt vanuit open-forms met kenmerk 'f8534f13-0669-4d4d-a364-6b6c4ad3d243'. fakeZaakToelichting",
-            "uiterlijkeEinddatumAfdoening": "${json-unit.any-string}",
-            "verlengingReden" : "fakeReason",
-            "vertrouwelijkheidaanduiding": "openbaar",
-            "zaaktype": "Melding evenement organiseren behandelen"
+          {
+            "equalToJson":{
+              "SmartDocument":{
+                "Selection":{
+                  "TemplateGroup":"root",
+                  "Template":"root template 1",
+                  "FixedValues":""
+                },
+                "Variables":{
+                  "OutputFormats":[
+                    {
+                      "OutputFormat":"docx"
+                    }
+                  ],
+                  "RedirectMethod":"POST",
+                  "RedirectUrl":"${json-unit.any-string}"
+                }
+              },
+              "data":{
+                "gebruiker":{
+                  "id":"testuser1",
+                  "naam":"Test User1 Špëçîâl Characters"
+                },
+                "taak":{
+                  "naam":"User details"
+                },
+                "zaak":{
+                  "communicatiekanaal":"fakeCommunicatiekanaal1",
+                  "groep":"Test group A",
+                  "identificatie":"ZAAK-2000-0000000004",
+                  "omschrijving":"fakeOmschrijving",
+                  "registratiedatum":"${json-unit.any-string}",
+                  "startdatum":"01-01-2000",
+                  "toelichting":"null",
+                  "uiterlijkeEinddatumAfdoening":"31-01-2000",
+                  "vertrouwelijkheidaanduiding":"openbaar",
+                  "zaaktype":"BPMN test zaaktype"
+                }
+              }
+            },
+            "ignoreArrayOrder":true
           }
-        }
-      },
-      "ignoreArrayOrder": true
-    } ]
+        ]
+      }
+    ]
   },
-  "response": {
-    "status": 200,
-    "body": "{\"ticket\": \"fakeSmartdocumentsTicketID\"}",
-    "headers": {
-      "Content-Type": "application/json"
+  "response":{
```

Comment:
I thought formatting of all JSON files was covered by Spotless?

Reply:
Maybe Spotless is no longer working as it used to be?

Reply:
Nope - now just `"src/**/*.json"`

Reply:
Created a PR to fix this